app/assets/javascripts/admin/modules/document-history.js

```diff
@@ -0,0 +1,45 @@
+window.GOVUK = window.GOVUK || {}
+window.GOVUK.Modules = window.GOVUK.Modules || {};
+
+(function (Modules) {
+  function DocumentHistory (module) {
+    this.module = module
+  }
+
+  DocumentHistory.prototype.init = function () {
+    this.setupEventListeners()
+  }
+
+  DocumentHistory.prototype.setupEventListeners = function () {
+    var documentHistoryTab = document.querySelector('#history_tab')
+
+    document.addEventListener('click', function (e) {
+      var newerLink = document.querySelector('.app-component-document-history-tab__newer-pagination-link')
+      var olderLink = document.querySelector('.app-component-document-history-tab__older-pagination-link')
+
+      if (newerLink && newerLink.contains(event.target)) {
+        e.preventDefault()
+        // eslint-disable-next-line
+        Rails.ajax({
```

Comment:
I would avoid using rails ajax and stick to using native `fetch` or  something like [this](https://github.com/alphagov/whitehall/blob/main/app/assets/javascripts/components/govspeak-editor.js#L21)